node/src/components/consensus/era_supervisor.rs

```diff
@@ -888,26 +943,14 @@ where
 }
 
 /// Computes the instance ID for an era, given the state root hash, block height and chainspec.
-fn instance_id(
-    protocol_config: &ProtocolConfig,
-    state_root_hash: Digest,
-    block_height: u64,
-) -> Digest {
+fn instance_id(protocol_config: &ProtocolConfig, state_root_hash: Digest) -> Digest {
     let mut result = [0; Digest::LENGTH];
     let mut hasher = VarBlake2b::new(Digest::LENGTH).expect("should create hasher");
 
     hasher.update(&protocol_config.name);
     hasher.update(protocol_config.timestamp.millis().to_le_bytes());
     hasher.update(state_root_hash);
 
-    for upgrade_point in protocol_config
-        .upgrades
-        .iter()
-        .take_while(|up| up.activation_point.height <= block_height)
-    {
-        hasher.update(upgrade_point.activation_point.height.to_le_bytes());
-    }
-
```

Comment:
We were including activation points here so that the instances for the same era ID of which one is upgraded and the other isn't would be incompatible. We're losing this property now. This probably won't be a big problem in practice, but I'm not sure. cc @afck

Reply:
It would be good to have different instance IDs. Otherwise a validator who missed the upgrade would start creating invalid messages with the right instance ID, and if they then later do the upgrade, they might equivocate.
(Our latest unit file should still prevent that… but then they would probably be mute for the whole era — because they'll never download what they think was their latest unit —, and get evicted.)

Reply:
Sure, but the `ProtocolConfig` won't have a list of activation points any more, just the single one relating to the current version.

Seems like it would suffice to use either the current protocol version instead or the current activation point?

Reply:
Right, that's true.
And we don't need the previous protocol versions because we don't instantiate the eras before the upgrade anyway. :+1:

Reply:
current protocol version